docs: adapt Documentation with new docarray at center (#5941)
jina-ai/jina: docs/cloud-nativeness/kubernetes.md

    def __init__(
        self,
        pretrained_model_name_or_path: str = 'openai/clip-vit-base-patch32',
        device: str = 'cpu',
        *args,
        **kwargs,
    ):
        super().__init__(*args, **kwargs)
        self.device = device
        self.pretrained_model_name_or_path = pretrained_model_name_or_path
        self.tokenizer = CLIPTokenizer.from_pretrained(self.pretrained_model_name_or_path)
        self.model = CLIPModel.from_pretrained(self.pretrained_model_name_or_path)

        self.model.eval().to(device)

    def _tokenize_texts(self, texts):
        x = self.tokenizer(
            texts,
            max_length=77,
            padding='longest',
            truncation=True,
            return_tensors='pt',
        )
        return {k: v.to(self.device) for k, v in x.items()}

    @requests
    def encode(self, docs: DocList[TextDoc], **kwargs) -> DocList[TextDoc]:
        with torch.inference_mode():
            input_tokens = self._tokenize_texts(docs.text)
            embeddings = self.model.get_text_features(**input_tokens).cpu().numpy()
            for doc, embedding in zip(docs, embeddings):
                doc.embedding = embedding

>         return docs

Author: I did not have yet the chance to test this, could be

To understand why you need to install a service mesh like Linkerd refer to this  {ref}`section <service-mesh-k8s>`

(build-containerize-for-k8s)=
## Build and containerize your Executors

> First of all we need to build the Executors we are going to use and containerize them {ref}`manually <dockerize-exec>` or leveraging {ref}`Executor Hub <jina-hub>`. In this example,

Reviewer: ```suggestion
First, we need to build the Executors that we are going to use and containerize them {ref}`manually <dockerize-exec>` or by leveraging {ref}`Executor Hub <jina-hub>`. In this example,
```
